### PR TITLE
Inaccurate screen reader alert

### PR DESCRIPTION
**Pull request by alflennik**
See https://github.com/w3c/aria-at-app/issues/797. We occasionally use aria-live to instruct screen readers to speak feedback to the user, however in one case the aria-live element could also be traversed by a screen readers, which was unnerving and not a best practice, and furthermore the information being spoken was inaccurate. This PR uses the more polished aria-live provider element which is currently in use elsewhere in the project and makes the information being spoken more helpful.

### CODE REVIEW

File: client/index.js [hunk 1]
```diff
@@ -5,13 +5,16 @@ import { BrowserRouter } from 'react-router-dom';
...
+import { AriaLiveRegionProvider } from './components/providers/AriaLiveRegionProvider';
 
 const container = document.getElementById('root');
 const root = createRoot(container);
 root.render(
     <GraphQLProvider>
-        <BrowserRouter>
-            <App />
-        </BrowserRouter>
+        <AriaLiveRegionProvider>
```
Comment (howard-e) [inline client/index.js:14]: Good change!

**Review by howard-e — APPROVED** [review]
LGTM! Also checked that the changes included for the DataManagement table sticks to the current behavior.

This should also make working with `AriaLiveRegionProvider` much easier in the future

**Comment by alflennik** [comment]
Big thanks @evmiguel for catching an issue with the "checked / unchecked" speech, could you take another look and approve if it looks good?

File: client/components/TestQueue/AssignTesterDropdown/index.jsx [hunk 2]
```diff
@@ -150,24 +149,31 @@ const AssignTesterDropdown = ({
...
-                                        );
+                                        const updatedIsAssigned =
+                                            !testerIsAssigned;
                                         setAlertMessage(
-                                            `${tester.username} has been ${
-                                                classname.includes('not')
-                                                    ? 'assigned to'
-                                                    : 'removed from'
-                                            } this test run.`
+                                            `${username} ${
```
Comment (evmiguel) [inline client/components/TestQueue/AssignTesterDropdown/index.jsx:157]: I can confirm that this feature works as expected.

File: client/components/TestQueue/AssignTesterDropdown/index.jsx [hunk 3]
```diff
@@ -150,24 +149,31 @@ const AssignTesterDropdown = ({
...
                                         );
+                                        await toggleTesterAssign(username);
                                         await onChange();
                                     }}
-                                    aria-checked={isTesterAssigned}
                                 >
                                     {icon && <FontAwesomeIcon icon={icon} />}
-                                    <span className={classname}>
+                                    <span className="sr-only">{`${username} ${
+                                        testerIsAssigned
```
Comment (evmiguel) [inline client/components/TestQueue/AssignTesterDropdown/index.jsx:169]: I can confirm that this feature works as expected.

**Review by evmiguel — APPROVED** [review]
Great work @alflennik! I can confirm that the AT message works well.

**Pull request merged**